optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/valuerange/descriptor/AbstractFromPropertyValueRangeDescriptor.java

```diff
@@ -108,6 +109,11 @@ private void processValueRangeProviderAnnotation(ValueRangeProvider valueRangePr
                 // with the variableDescriptor's generic type's type arguments
                 typeArgument = ((ParameterizedType) typeArgument).getRawType();
             }
+
+            if(typeArgument instanceof WildcardType ) {
```

Comment:
Our code style prescribes a space after `if`, and no whitespace before the closing parenthesis.
Also, it would be nice if you could provide a reproducing test.

Reply:
I have fixed the whitespace feedback on the if block. Please let me know if it needs any changes. Will it be possible to share code style guidelines or IDE formatter config if they are publically available, it will help any future commits from me.

I have also added a failing test case that passes after applying the fix. Please let me know if it helps.